gradle.properties

```diff
@@ -7,7 +7,7 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 100
+cordaApiRevision = 102
```

Comment:
Have you skipped a version here or is this a case of needing to update the PR?

Reply:
I haven't rebased (due to the version in between making my code uncompiled due to the usual api/runtime mismatch), but this should just merge straight in.

https://github.com/corda/corda-api/blob/release/os/5.0/gradle.properties#L10

Reply:
fixed